backend/api/sponsors/types.py

```diff
@@ -28,3 +31,8 @@ class SponsorsByLevel:
     level: str
     sponsors: typing.List[Sponsor]
     highlight_color: typing.Optional[str]
+
+    @classmethod
+    def from_model(cls, level: SponsorLevelModel) -> SponsorsByLevel:
+        sponsors = [sponsor for sponsor in level.sponsors.all()]
+        return cls(level.name, sponsors, level.highlight_color)
```

Comment:
I feel like this would be safer:

```suggestion
        return cls(name=level.name, sponsors=sponsors, highlight_color=level.highlight_color)
```